data/abilities/collection/02de522f-7e0a-4544-8afc-0c195f400f5f.yml

```diff
@@ -12,7 +12,13 @@
       sh:
         command: |
           pip install stormssh && storm list
+        parsers:
+          plugins.stockpile.app.parsers.ssh:
```

Comment:
this seems like a broad name for the parser and could be ambiguous if this is for commands, keys, known hosts etc ...

can leave as is for now but something to think about